components/automate-ui/src/app/page-components/profile/profile.component.html

```diff
@@ -27,8 +27,9 @@
         </li>
       </app-authorized>
       <li class="dropdown-list-item">
-        <span class="version">Version: {{ buildVersion }}</span>
-        <ul class="dropdown-sub-list">
+        <span class="version">Version: 2</span>
+        <span class="build">Build: {{ buildVersion }}</span>
+		<ul class="dropdown-sub-list">
```

Comment:
Indentation looks a bit wonky, doesn't it?

Reply:
it does. my editor does not get along with ui code. or, well, i guess i should say i haven't taken the time to set up my editor so that it does the right thing when it auto-detects the language in the file